proto/reaction.proto

```diff
@@ -141,6 +141,11 @@ message Compound {
     Moles moles = 3;
     Volume volume = 4;
   }
+  // Whether the volume measurement refers to the pure substance or to the
+  // total volume of the reaction input. An example of when this field should
+  // be TRUE is when stock solutions are prepared by adding solvent to a
+  // volumetric flask already containing solute(s).
+  Boolean.BooleanValue volume_includes_solutes = 14;
```

Comment:
I'm ok with this, but some possible alternatives:

* `is_final_volume`
* `is_total_volume`

Reply:
For clarity over brevity, let's stick to `volume_includes_solutes`. This also reinforces this field's relevance to stock solution preparation